lib/index.js

```diff
@@ -533,8 +546,6 @@ Socket.prototype._flush = function() {
 
         throw sendError;
       }
-
-      flags = this._ioevents;
```

Comment:
yea i agree with that because we're doing both `send()` and `recv()` functionality inside flush